bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,2078 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="luxtronikheatpump"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="Temperatur_TVL">
+		<item-type>Number:Temperature</item-type>
+		<label>Flow temperature heating circuit</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars.

Reply:
I've capitalized them for now. might be hard to shorten some of the labels. Will need to think about them a bit and see if some better names will come to my mind...

Reply:
There are still a lot channel labels with more than 25 chars. But I'm not sure how to shorten them without changing the meaning. Is it ok to use abbreviations where possible?

Reply:
I think it's ok to abbreviate temperature etc., but I wouldn't use abbreviations at any price.